extensions/wikia/Email/EmailController.class.php

```diff
@@ -644,4 +644,13 @@ private static function getLegendName() {
 
 		return $legendName;
 	}
+
+	/**
+	 * A wrapper for wfMessage() which removes the possibility of messages being overridden in the MediaWiki namespace
+	 *
+	 * @return Message
+	 */
+	protected function getMessage() {
+		return wfMessage( func_get_args() )->useDatabase( false );
```

Comment:
@mklucsarits were you able to get this to work before? I didn't test this, but when I just tried merging it with my code none of the messages which have parameters are coming through.

I think the problem is that wfMessage always expects the first argument to be the message string. By calling `func_get_args()` and passing that result to wfMessage, the first and only argument now becomes an array which is causing problems in [here](https://github.com/Wikia/app/blob/dev/includes/GlobalFunctions.php#L1352-L1358).